### DISCUSSION TITLE
Deploys Having Detrimental Effect On Other Services

### DISCUSSION THREAD

**Discussion by darrylmorley** [caprover/caprover#2080] [Q&A] [answered]
Hi,

I'm running several Nextjs sites on my Digital Ocean server. When deploying an update to one of my sites my site monitor triggers warnings that the other sites are unreachable, they are still live just responding very slowly due to the background build. Maybe it would be better to build locally, and then deploy? Can that me achieved with caprover and are there any gotcha's to look out for?

Thanks

**Comment by githubsaturn** [comment, marked as answer]
Better than that, you can [build on a CI/CD server](https://caprover.com/docs/ci-cd-integration.html#:~:text=often%20you%20realize%20that%20the%20build%20process%20is%20very%20heavy.%20In%20fact%2C%20in%20many%20cases%2C%20it%27s%20heavier%20than%20the%20load%20on%20your%20app%20itself)


Alternatively, you can build your static site locally and just push the built files and have CapRover only create the simple Docker image, here is guide for that:
https://caprover.com/docs/recipe-deploy-create-react-app.html

**Reply by darrylmorley** [reply]
Thanks for the link 👍
